draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1041,20 +1080,21 @@ pass the following result on to the application for further processing:
 
 <{{examples/complex_ekyc/verified_contents.json}}
 
-## Example 4 - W3C Verifiable Credentials Data Model (work in progress)
+## Example 4a - W3C Verifiable Credentials Data Model v2.0, not using JSON-LD
 
 This example illustrates how the artifacts defined in this specification can be
-represented using W3C Verifiable Credentials Data Model as defined in
-[@VC_DATA].
+represented as W3C Verifiable Credentials Data Model v2.0 [@VC_DATA_v2.0] without using JSON-LD.
+It uses a content type `credential-claims-set+json` defined in [@VC_JWT], Section 3.
```

Comment:
I think the text should also define the `typ` value. Suggest `vc+sd-jwt`. I also suggest to explicitly mention the header name `cty` for the content type.

Reply:
yes, `typ` should be defined for sd-jwt-vc, I added for clarity, but I strongly believe then it should be in the main text and not the examples that are **non-normative**.

Reply:
`cty` clarified.

Reply:
Non-normative examples should not be using (or pseudo defining) nonexistent media types.  

`vc+sd-jwt` isn't a registered media type nor is there text anywhere that's intending to define/register it. It also suggests a structured-suffix media type of `+sd-jwt` which similarly doesn't exist. Those maybe things that'd be desirable to define/register but not just by putting them in examples. That's a somewhat different direction than what's being proposed in #229 too.